src/openklant/components/klantinteracties/models/partijen.py

```diff
@@ -0,0 +1,157 @@
+import uuid
+
+from django.core.validators import validate_integer
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from .constants import SoortPartij
+from .digitaal_adres import DigitaalAdres
+from .klantcontacten import Betrokkene
+from .mixins import BezoekAdresMixin, ContactNaamMixin, CorrespondentieAdresMixin
+
+
+class Partij(BezoekAdresMixin, CorrespondentieAdresMixin):
+    uuid = models.UUIDField(
+        primary_key=True,
+        default=uuid.uuid4,
+        help_text=_("Unieke (technische) identificatiecode van de partij."),
+    )
+    betrokkene = models.ForeignKey(
+        Betrokkene,
+        on_delete=models.CASCADE,
+        verbose_name=_("Betrokkene"),
+        related_name="partijen",
+        help_text=_("'Betrokkene bij klantcontact' was 'Partij'"),
+        null=True,
+    )
+    digitaal_adres = models.ForeignKey(
+        DigitaalAdres,
+        on_delete=models.CASCADE,
+        verbose_name=_("Digitaal adres"),
+        related_name="digitale_adressen",
+        help_text=_("'Digitaal Adres' was 'Partij'"),
+        null=True,
+    )
```

Comment:
```suggestion
    )
    voorkeurs_digitaal_adres = models.ForeignKey(
        DigitaalAdres,
        on_delete=models.CASCADE,
        verbose_name=_("Voorkeurs digitaal adres"),
        help_text=_("'Partij' gaf voorkeur aan voor contact via 'Digitaal adres'"),
        null=True,
    )
```

Dit is nieuw in IM 6